pkgs/top-level/ocaml-packages.nix

```diff
@@ -45,6 +45,8 @@ let
 
     base64 = callPackage ../development/ocaml-modules/base64 { };
 
+    bibtex2html = callPackage ../tools/misc/bibtex2html { };
+
     bolt = callPackage ../development/ocaml-modules/bolt { };
```

Comment:
This package does not build an OCaml library: it does not belong to `ocaml-packages.nix`.